android/KMEA/app/src/main/java/com/tavultesoft/kmea/JSONParser.java

```diff
@@ -30,11 +30,25 @@ public JSONObject getJSONObjectFromUrl(String urlStr) {
     HttpURLConnection urlConnection = null;
 
     try {
+      // TODO: Refactor urlConnection to utility method
       URL url = new URL(urlStr);
+      HttpURLConnection.setFollowRedirects(false);
```

Comment:
I take it that using `setFollowRedirects(true)` doesn't follow properly, hence our doing it manually in code below?

Reply:
From what I'm reading, `setFollowRedirects` defaults to `true`, but only follows if the protocol remains the same.